webpack.config.js

```diff
@@ -320,9 +323,10 @@ let appConfig = {
 
     new SentryInstrumentation(),
 
-    ...(!IS_CI
+    ...(IS_DEV_ENV
```

Comment:
In line with the one above:
```suggestion
    ...(SHOULD_FORK_TS
```